### ISSUE TITLE
Шаблонизация на сервере

### ISSUE THREAD

**Issue by minya** [bem/bem-forum-content-ru#246]
Всем привет!
Подскажите, где посмотреть пример серверной шаблонизации? Чтоб одни и те же блоки можно было рендерить и на сервере и на клиенте.

**Comment by kuflash** [comment]
@minya, вот есть такая [штука](https://github.com/apsavin/bnsf) от @apsavin Там в описании есть stub проект и пример использования

**Comment by tadatuta** [comment]
@minya в простом случае достаточно
```js
require('.desktop.bundles/index/index.bemhtml.js').BEMHTML.apply({ block: 'page' }); // вернется строка
```

**Comment by minya** [comment]
@tadatuta
То есть, мне надо сначала сделать фейковую страницу, на которую накидать все блоки, которые я буду использовать. Потом напустить на неё сборщик. Он сгенерит desktop.bundles/index/index.bemhtml.js. И его-то я и смогу использовать для генерации страницы. Верно?
Подозреваю, что можно сборку настроить. Так, чтоб ей только deps-ы указать.

**Comment by Guria** [comment]
Это называется сборка по bemdecl файлу. С телефона ссылки не готов кинуть.

**Comment by tadatuta** [comment]
@minya да, все верно, в bemjson (либо bemdecl) можно оставить всего один «стартовый» блок, а все остальные попадут в сборку по депсам.

а можно просто перечислить все сущности в bemdecl либо deps.js-формате. `bem-tools` будет использовать bemdecl.js либо deps.js автоматически, если не окажется bemjson.js. а для `ENB` потребуется немного поправить конфиг сборки (см. https://github.com/enb-bem/enb-bem-techs/blob/master/docs/build-bundle.md).

**Comment by minya** [comment]
Да. Кажется, бандл -- это то, что нужно.
